libraries/AP_NavEKF3/AP_NavEKF3_Outputs.cpp

```diff
@@ -51,6 +51,16 @@ float NavEKF3_core::errorScore() const
     return score;
 }
 
+void NavEKF3_core::updateCoreError()
+{
+    runningError += NavEKF3_core::errorScore();
```

Comment:
this looks like runningError will always increase. I think we need a number that is on the same scale throughout the flight, so we can have meaningful thresholds.
If you leave the log up in AP_NavEKF3.cpp then I think you could accumulate the difference between the error score of each lane and the error score of the primary. That number would always be zero for the primary, and would grow as +ve or -ve number for lanes that are worse or better than the primary.

Reply:
Yes, the unbounded error is an issue, I was thinking about some normalisation to constrain it in a range but could not figure out logically. I'll modify this to accumulating differences with respect to the primary. Also when we switch lanes, should I reset all the errors by making the new primary's error 0 or continue with the existing, which also is fine as error would still accumulate in the respective lanes, just there would be an initial offset which I don't know if it would create any problems.

Reply:
I think reset all to zero on switch